components/ratgdo/ratgdo.cpp

```diff
@@ -222,10 +233,47 @@ namespace ratgdo {
                 this->send_command(Command::GET_STATUS);
             }
             ESP_LOGD(TAG, "Motion: %s", MotionState_to_string(*this->motion_state));
-        } else if (cmd == Command::SET_TTC) {
+        } else if (cmd == Command::TTC_SET_DURATION) {
             auto seconds = (byte1 << 8) | byte2;
-            ESP_LOGD(TAG, "Time to close (TTC): %ds", seconds);
-        }
+            ESP_LOGD(TAG, "Time to close (TTC) update request: %ds", seconds);
+        } else if (cmd == Command::TTC_DURATION) {
+            auto seconds = (byte1 << 8) | byte2;
+            ESP_LOGD(TAG, "Time to close (TTC) set to: %ds", seconds);
+            if (seconds == 60) {
```

Comment:
It looks like this block could be combined a bit

Reply:
Combined

https://github.com/ESPHome-RATGDO/esphome-ratgdo/pull/43/files#diff-8ec49aa77fc5c9af707edd16a08e03e3673aa20fc288a014c9d117f0edc5c2a3R245-R250